### PR TITLE
[ADD] create the admin site

### PR DESCRIPTION
**Pull request by kate-brushkova**

### CODE REVIEW

**Comment by kate-brushkova** [comment]
@o-diatlenko please review

File: polls/models.py [hunk 1]
```diff
@@ -1,3 +1,24 @@
+import datetime
+
 from django.db import models
+from django.utils import timezone
+
+
+class Question(models.Model):
+    question_text = models.CharField(max_length=200)
+    pub_date = models.DateTimeField('date published')
```
Comment (o-diatlenko) [inline polls/models.py]: add key to the field

File: polls/models.py [hunk 2]
```diff
@@ -1,3 +1,24 @@
...
+
+    def __str__(self):
+        return self.question_text
+
+    def was_published_recently(self):
+        return self.pub_date >= (timezone.now() - datetime.timedelta(days=1)).timestamp()
+
+
+class Choice(models.Model):
+    question = models.ForeignKey(Question, on_delete=models.CASCADE)
```
Comment (o-diatlenko) [inline polls/models.py]: add key too

File: polls/models.py (hunk 1 above)
Comment (kate-brushkova) [reply polls/models.py]: done

**Review by o-diatlenko — APPROVED** [review]

**Comment by o-diatlenko** [comment]
well done

**Pull request merged**